src/ml/neural_net/style_transfer/mps_style_transfer_backend.hpp

```diff
@@ -50,4 +51,4 @@ class EXPORT mps_style_transfer : public turi::neural_net::model_backend {
 
 #endif // #ifdef HAS_MACOS_10_15
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
i think all of our supported compilers should support `#pragma once`? should we incrementally move to that as we touch headers?

Reply:
I wasn't previously familiar with `#pragma once`, but it seems reasonable. I am not aware that any of our build processes copy headers in a way that would be problematic here. I'll kick off a build to verify that the compiler we use for our releases does the right thing